src/foundry/client/core/keybindings.d.mts

```diff
@@ -149,16 +149,18 @@ declare global {
      */
     static _compareActions(a: KeybindingAction, b: KeybindingAction): number;
 
-    /** Register core keybindings */
-    protected _registerCoreKeybindings(): void;
+    /**
+     * Register core keybindings.
+     * @param view           - The active game view
+     */
+    protected _registerCoreKeybindings(view: string): void;
```

Comment:
I wonder if we should enumerate the possible game views somewhere. Probably in `Game`? Might already be done.

Reply:
There is a `Game.View`, so I can update this.  Strangely, it's defined as `"game" | "stream"`, but the comment on the only place that appears to use it says `Game views include: join, setup, players, license, game, stream`

Reply:
The other views aren't relevant to user packages — e.g. the setup screen doesn't load third party code.